editor/src/components/canvas/controls/new-canvas-controls.tsx

```diff
@@ -227,11 +230,11 @@ export const selectElementsThatRespectLayout = createSelector(
     openImports: Imports,
     openFilePath: string | null,
     rootComponents: UtopiaJSXComponent[],
-    jsxMetadataKILLME: JSXMetadata,
+    jsxMetadataKILLME: ElementInstanceMetadataMap,
```

Comment:
Should this still be suffixed with `KILLME`?

Reply:
Ha probably not. I shall remove the `KILLME`

Reply:
Done :+1: